src/main/java/eu/yals/ui/err/ServerErrorView.java

```diff
@@ -0,0 +1,199 @@
+        if (StringUtils.isNotBlank(yalsError.getTimeStamp())) {
+            when.setText(yalsError.getTimeStamp());
+        }
+        if (StringUtils.isNotBlank(yalsError.getMessageToUser())) {
+            what.setText(yalsError.getMessageToUser());
+        }
+        triggerPanelsBasedOnTextInside(userMessagePanel, timeStampPanel);
+
+        if (appUtils.isDevelopmentModeActivated() || appUtils.hasDevHeader()) {
+            if (StringUtils.isNotBlank(yalsError.getTechMessage())) {
+                String techMessage = formatTechMessage(yalsError.getTechMessage());
+
+                appUtils.pasteHtmlToComponent(techMessage, messageSpan);
+            }
+            if (yalsError.getRawException() != null) {
+                String traceMessage = AppUtils.stackTraceToString(yalsError.getRawException());
+                String trace = formatTrace(traceMessage);
+
+                appUtils.pasteHtmlToComponent(trace, traceSpan);
+            }
+            triggerPanelsBasedOnTextInside(techMessagePanel, tracePanel);
+        }
+    }
+
+    /**
+     * Disables given accordion panels if they are empty, enables otherwise
+     *
+     * @param panels initialised accordion panels
+     */
+    private void triggerPanelsBasedOnTextInside(AccordionPanel... panels) {
+        for (AccordionPanel panel : panels) {
+            Optional<Component> elementWithText = panel.getContent().findFirst();
+            if (elementWithText.isPresent()) {
+                Span spanWithText = (Span) elementWithText.get();
+                boolean ifSpanHasText = StringUtils.isNotBlank(spanWithText.getText()) ||
+                        StringUtils.isNotBlank(spanWithText.getElement().getProperty(HTML_MODE));
+
+                panel.setEnabled(ifSpanHasText);
+            } else {
+                panel.setEnabled(false);
+            }
+        }
+    }
+
+    private String formatTechMessage(String techMessage) {
+        return techMessage
+                .replaceAll(App.NEW_LINE, App.WEB_NEW_LINE)
+                .replaceAll(";", App.WEB_NEW_LINE)
+                .replaceAll("nested exception is", "&emsp;->");
+    }
+
+    private String formatTrace(String traceMessage) {
+        return traceMessage.replaceAll(App.NEW_LINE, App.WEB_NEW_LINE)
+                .replaceAll("at ", "&emsp;&emsp;at ");
+    }
+
+    /**
+     * EntryPoint from {@link YalsErrorController}
+     *
+     * @param event     Vaadin Event with location, payload
+     * @param parameter string goes after errors/500. We ignore it, because we use queryParams instead
+     */
+    @Override
+    public void setParameter(BeforeEvent event, @OptionalParameter String parameter) {
+        YalsError yalsError = errorUtils.getYalsErrorFromEvent(event);
+        if (Objects.isNull(yalsError)) {
+            event.rerouteToError(GeneralServerException.class, Integer.toString(500));
+            return;
+        }
+
+        switch (yalsError.getHttpStatus()) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A switch statement does not contain a break](https://app.codacy.com/gh/yadevee/yals/pullRequest?prid=5392475)